modules/openapi-generator/src/main/resources/dart/libraries/dio/pubspec.mustache

```diff
@@ -1,6 +1,6 @@
 name: {{pubName}}
 version: {{pubVersion}}
-description: {{pubDescription}}
+description: {{{pubDescription}}}
```

Comment:
```suggestion
description: '{{{pubDescription}}}'
```

Reply:
Actually, this would break any package with a single quote as the single quote is not escaped ...

Reply:
meant to do double quotes, but would also be the same problem, I think it's best to just remove quotes altogether as you said

Reply:
I'd propose that we remove the quotes from the dart2 package!

The other alternative would be to use `>-` as YAML files do not support escaping strings
https://stackoverflow.com/a/37432679